spec/graphql/mutations/user_store_item/create_user_store_item_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+module Mutations
+  module UserStoreItems
+    RSpec.describe CreateUserStoreItem, type: :request do
+      describe '.resolve' do
+        let!(:user) {create(:user)}
+        let!(:store) {create(:store)}
+        let!(:item) {create(:item)}
+        let!(:store_item) {StoreItem.create!(store: store, item: item, price: "$1.99")}
```

Comment:
Do we want the price to be a float? That's the datatype in the database.

Reply:
whoops yeah thank you for catching that!